index.html

```diff
@@ -0,0 +1,105 @@
+  </head>
+
+  <body>
+    <section id="main-container">
+      <div class="photo-container">
+        <p class="text">
+          <strong id="big-title">Shipping</strong><br />Project
+        </p>
+        <img class="image" src="src/shopping.png" alt="Shipping Image" />
+      </div>
+
+      <div class="textbox-container">
+        <div class="user-side">
+          <p class="border-bottom title">Kullanıcı Bilgileri</p>
+          <input
+            type="text"
+            class="height-textbox isim"
+            placeholder="İsim Soyisim"
+          />
+          <input
+            type="text"
+            class="height-textbox telefon"
+            placeholder="Telefon"
+          />
+          <textarea rows="5" class="adres" placeholder="Adres"></textarea>
+        </div>
+
+        <div class="invoice-side">
+          <div class="border-bottom" id="invoice-side-title">
+            <p class="title">Fatura Bilgileri</p>
+            <div style="display: flex; flex-direction: row">
+              <label
+                for="vehicle1"
+                style="font-size: 11px; padding-right: 5px; color: #686868"
+              >
+                Fatura bilgisi olarak üsttekileri kullan</label
+              >
+              <input
+                type="checkbox"
+                class="checkbox-round check"
+                id="vehicle1"
+                name="vehicle1"
+                value="Bike"
+              />
+            </div>
+          </div>
+          <input
+            type="text"
+            class="height-textbox fatura"
+            placeholder="Fatura Sahibi"
+          />
+          <input
+            type="text"
+            class="height-textbox"
+            placeholder="T.C. / Vergi Numarası"
+          />
+          <textarea
+            rows="5"
+            class="fatura-adresi"
+            placeholder="Fatura Adresi"
+          ></textarea>
+        </div>
+
+        <button class="btn">Sipariş Tamamla</button>
+      </div>
+    </section>
+    <div class="modal hidden">
+      <button class="close-modal">&times;</button>
+      <h1 class="modal-isim">Siparişiniz tamamlandı.</h1>
+      <p class="modal-icerik">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quaerat ab
+        quia repellendus soluta reiciendis, inventore recusandae vel maiores.
+        Incidunt labore corporis modi voluptate. Ab dolorem officiis suscipit
+        eveniet eius pariatur!
+      </p>
+    </div>
+    <div class="overlay hidden"></div>
+    <footer>
+      <p>
+        Tasarım : <a style="text-decoration: none; color: white" href="https://github.com/YunusInoluk">Yunus</a>
+        HTML / CSS : <a style="text-decoration: none; color: white" href="https://github.com/hacergizem">Gizem</a> - 
+        <a style="text-decoration: none; color: white" href="https://github.com/sinandmr">Sinan</S></a>
+        <b></b>
+      </p>
+    </footer>
+  </body>
+  <script src="app.js"></script>
```

Comment:
app.js file is missing. This line can be removed or app.js file can be added. If you prefer to add app.js file, show an alert when the page is loaded and write something similar to this text:

"Projenin sadece tasarımı yayınlanmıştır. Javascript kodları yazılmamıştır"